cmd/charm/charmcmd/listresources.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the GPLv3, see LICENCE file for details.
+
+package charmcmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"text/tabwriter"
+
+	"github.com/juju/cmd"
+	"gopkg.in/errgo.v1"
+	"gopkg.in/juju/charm.v6-unstable"
+	"gopkg.in/juju/charmrepo.v2-unstable/csclient/params"
+	"launchpad.net/gnuflag"
+)
+
+var info = cmd.Info{
+	Name:    "list-resources",
+	Args:    "<charm> [--channel <channel>]",
+	Purpose: "display the resources for a charm in the charm store",
+	Doc: `
+This command will report the resources for a charm in the charm store.
+
+<charm> can be a charm URL, or an unambiguously condensed form of it,
+just like the deploy command. So the following forms will be accepted:
```

Comment:
I don't think we should reference the deploy command, since that's a juju command not a charm command.